src/main/ts/Part2Ex2ArrayFunctions.ts

```diff
@@ -83,12 +83,10 @@ export const evens = (xs: number[]): number[] =>
 
 // TODO: Write a function that returns all the frogs that ribbit
 // TODO: Run the provided test to check your answer.
-export const ribbitting = (frogs: Frog[]): Frog[] =>
-  [];
+export const ribbitting = (frogs: Frog[]): Frog[] => Arr.filter(frogs, (f: Frog) => f.ribbits === true)
```

Comment:
```suggestion
export const ribbitting = (frogs: Frog[]): Frog[] => Arr.filter(frogs, (f: Frog) => f.ribbits)
```
The `=== true` here is redundant since it's a non-optional boolean, so we're normally just do something like the above instead.